changelogs/fragments/57801-openssh_keypair-fix-make-regenerating-valid-files-possible.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - openssh_keypair - make regeneration of valid keypairs with the ```force``` option possible, add better handling for invalid files
```

That's one backtick too many :)
```suggestion
  - openssh_keypair - make regeneration of valid keypairs with the ``force`` option possible, add better handling for invalid files
```